org.nuspell.Nuspell.json

```
@@ -0,0 +1,64 @@
+{
+  "app-id": "org.nuspell.Nuspell",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
```

Update runtime to 19.08
```suggestion
  "runtime-version": "19.08",
```

---

Thanks